docs/developer-guide.md

```diff
@@ -3,8 +3,8 @@
 ## Develop
 
 - `npm run dev`. It uses ts-node to spin up the feed builder script.
-  - It will use `osmosnote.yaml` in the project root.
-- To test remote caching, enable/update the cache url option inside `osmosnote.yaml`.
+  - It will use `osmosfeed.yaml` in the project root.
```

Comment:
Not sure I change it in the right way

Reply:
Thank you and yes you did! Clearly I've been multi-tasking on several projects 😅